### PR TITLE
1169 add non static site initialisation example

### PR DESCRIPTION
**Pull request by vladb00mer**
!please delete this branch after merging!

### CODE REVIEW

File: jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import selenium.site.sections.LoginForm;
 
 @JSite("https://jdi-testing.github.io/jdi-light/")
-public class SiteJdi {
+public class StaticSiteJdi {
```
Comment (elv1s42) [inline jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java]: Codacy Issue found: [The utility class name 'StaticSiteJdi' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
- jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java  1
- jdi-light-examples/src/main/java/selenium/site/NonStaticSiteJdi.java  2
- jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java  2
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156&bid=15123869)

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package io.github.com;
+
+import com.epam.jdi.light.common.CheckTypes;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.XPath;
+import io.github.com.pages.*;
```
Comment (elv1s42) [inline jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.com.pages'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

File: jdi-light-examples/src/main/java/io/github/epam/EpamGithubStaticSite.java [hunk 3]
```diff
@@ -15,7 +15,7 @@
 import java.util.List;
 
 @JSite("https://jdi-testing.github.io/jdi-light/")
-public class EpamGithubSite {
+public class EpamGithubStaticSite {
```
Comment (elv1s42) [inline jdi-light-examples/src/main/java/io/github/epam/EpamGithubStaticSite.java]: Codacy Issue found: [The utility class name 'EpamGithubStaticSite' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

File: jdi-light-examples/src/main/java/selenium/site/NonStaticSiteJdi.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    @FindBy(css = ".sidebar-menu [ui=label]")
+    public WebList navigation;
+
+    @Step("Login")
+    public void login(User user) {
+        if (!userName.isDisplayed()) {
+            profilePhoto.click();
+            loginForm.submit(user, "enter");
+        }
+    }
```
Comment (elv1s42) [inline jdi-light-examples/src/main/java/selenium/site/NonStaticSiteJdi.java]: Codacy Issue found: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

File: jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java [hunk 5]
```diff
@@ -0,0 +1,64 @@
+package io.github.epam;
+
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.Frame;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import io.github.epam.pages.*;
```
Comment (elv1s42) [inline jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.epam.pages'](https://app.codacy.com/app/jdi-testing/jdi-light/pullRequest?prid=4482156)

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java (hunk 2 above)
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: this import used for private HomePage homePage;
all imports are used

File: jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java (hunk 5 above)
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/epam/EpamGithubNonStaticSite.java]: this import used for private HomePage homePage;
all imports are used

File: jdi-light-examples/src/main/java/io/github/epam/EpamGithubStaticSite.java (hunk 3 above)
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/io/github/epam/EpamGithubStaticSite.java]: class name EpamGithubStaticSite does match such mask

File: jdi-light-examples/src/main/java/selenium/site/NonStaticSiteJdi.java (hunk 4 above)
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/selenium/site/NonStaticSiteJdi.java]: unnecessary constructor removed with unnecessary class

File: jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java (hunk 1 above)
Comment (vladb00mer) [reply jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java]: it does

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           

Complexity increasing per file
==============================
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
- jdi-light-examples/src/main/java/selenium/site/StaticSiteJdi.java  2
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4482156&bid=15123869)

File: jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java (hunk 2 above)
Comment (elv1s42) [inline jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java]: Codacy Issue found: [Avoid unused imports such as 'io.github.com.pages'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4482156)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4482156&bid=15123869)

**Comment by elv1s42** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity increasing per file
==============================
- jdi-light-examples/src/test/java/io/github/epam/tests/nonstatic/NonStaticSiteTests.java  1
- jdi-light-examples/src/main/java/io/github/com/NonStaticSite.java  2
- jdi-light-examples/src/test/java/io/github/epam/NonStaticTestsInit.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/jdi-light/jdi-light/pullRequest?prid=4482156&bid=15123869)

**Pull request closed without merging**